mobile/status.go

```diff
@@ -1046,9 +1045,9 @@ func GetConnectionStringForBeingBootstrapped(configJSON string) string {
 		return makeJSONResponse(fmt.Errorf("no config given, PayloadSourceConfig is expected"))
 	}
 
-	statusBackend.SetLocalPairing(true)
+	statusBackend.LPM().SetPairing(true)
 	defer func() {
-		statusBackend.SetLocalPairing(false)
+		statusBackend.LPM().SetPairing(false)
```

Comment:
Just thought about it: does this make sense here?
We don't actually start the pairing here, only connection string is created. And it's definitely not finished when we get out of this function.